library/jenkins

```diff
@@ -2,6 +2,6 @@
 # maintainer: Michael Neale <mneale@cloudbees.com> (@michaelneale)
 # maintainer: Carlos Sanchez <csanchez@cloudbees.com> (@carlossg)
 
-latest: git://github.com/jenkinsci/jenkins-ci.org-docker@83ce6f6070f1670563a00d0f61d04edd62b78f4f
-1.625.3: git://github.com/jenkinsci/jenkins-ci.org-docker@83ce6f6070f1670563a00d0f61d04edd62b78f4f
```

Comment:
We used to have all versions here, but after discussion with Docker-library maintainer, they preferred we focus on latest LTS.
Please note the previous tag is still available on dockerhub, just not managed by docker-library scripts